packages/react/src/combobox/ComboboxBase/ComboboxBase.tsx

```diff
@@ -112,6 +115,15 @@ export function ComboboxBase<Option extends DefaultComboboxOption>({
 
 	const { id: labelId } = combobox.getLabelProps();
 
+	const handleOnBlur = (event: FocusEvent<HTMLInputElement>) => {
+		// If a user presses the Done button on the iOS keyboard, we shouldn't close the keyboard
+		if (isIos && !event.nativeEvent?.relatedTarget) {
+			// @ts-expect-error: Property 'preventDownshiftDefault' does not exist on type 'FocusEvent'
```

Comment:
Not sure if there's a way to somehow get the correct type from Downshift

Reply:
[Downshift don't seem to support this](https://github.com/downshift-js/downshift/issues/734#issuecomment-650546328). Probably best to leave this with the targeted solution you have rather than the suggested type-casting.